packages/core/README.md

```diff
@@ -236,6 +237,15 @@ When using the config file you can provide an array with `string`s
 
 You can *ignore* files by adding an exclamation mark (`!`) at the start of an expression.
 
+<a name="dryRunTimeoutMinutes"></a>
+### `dryRunTimeoutMinutes` [`number`]
+
+Default: `5`
```

Comment:
It is important that this line has 2 trailing whitespaces (or a `\`). See https://github.github.com/gfm/#hard-line-breaks

The same goes for the other 2 lines below.

Reply:
Ah good to know. :) Fixed it, though it seems strange that only the new options use the line break. Maybe this can be unified at some point (didn't want to include it here since this pr is only about the timeout paramter.)